python/lsst/pipe/base/_instrument.py

```diff
@@ -80,6 +81,10 @@ class Instrument(metaclass=ABCMeta):
     of the dataset type name, a tuple of dimension names, and the storage class
     name. If `None` the ingest system will use its default definition."""
 
+    day_obs_offset: int = 0
+    """Offset (in seconds) to apply when converting a ``day_obs`` value
+    to a timespan (`int`)."""
```

Comment:
This might be better as an abstract method or property so third-party `obs_` packages don't miss the need to set it and fill their databases with bad `day_obs` timespans before they find out.